packages/viewer/src/lib/types/frameCamera.ts

```diff
@@ -1,15 +1,113 @@
-import { Vector3 } from '@vertexvis/geometry';
+import { vertexvis } from '@vertexvis/frame-streaming-protos';
+import { Angle, Vector3 } from '@vertexvis/geometry';
 
-export interface FrameCamera {
+export interface PerspectiveFrameCamera {
   position: Vector3.Vector3;
   lookAt: Vector3.Vector3;
   up: Vector3.Vector3;
 }
 
-export function create(data: Partial<FrameCamera> = {}): FrameCamera {
+export interface OrthographicFrameCamera {
+  viewVector: Vector3.Vector3;
+  lookAt: Vector3.Vector3;
+  up: Vector3.Vector3;
+  fovHeight: number;
+}
+
+export type FrameCamera = PerspectiveFrameCamera | OrthographicFrameCamera;
+
+export function isPerspectiveFrameCamera(
+  camera: Partial<FrameCamera>
+): camera is PerspectiveFrameCamera {
+  return (camera as PerspectiveFrameCamera).position != null;
+}
+
+export function isOrthographicFrameCamera(
+  camera: Partial<FrameCamera>
+): camera is OrthographicFrameCamera {
+  const asOrtho = camera as OrthographicFrameCamera;
+  return asOrtho.viewVector != null && asOrtho.fovHeight != null;
+}
+
+export function createPerspective(
+  data: Partial<PerspectiveFrameCamera> = {}
+): PerspectiveFrameCamera {
   return {
-    position: data.position || Vector3.origin(),
+    position: data.position || Vector3.forward(),
     lookAt: data.lookAt || Vector3.origin(),
     up: data.up || Vector3.up(),
   };
 }
+
+export function createOrthographic(
+  data: Partial<OrthographicFrameCamera> = {}
+): OrthographicFrameCamera {
+  return {
+    viewVector: data.viewVector || Vector3.back(),
+    lookAt: data.lookAt || Vector3.origin(),
+    up: data.up || Vector3.up(),
+    fovHeight: data.fovHeight ?? 45,
+  };
+}
+
+export function toOrthographic(
+  data: PerspectiveFrameCamera
+): OrthographicFrameCamera {
+  const viewVector = Vector3.subtract(data.lookAt, data.position);
+
+  return {
+    viewVector,
+    up: data.up,
+    lookAt: data.lookAt,
+    fovHeight:
+      2 * Vector3.magnitude(viewVector) * Math.tan(Angle.toRadians(22.5)),
```

Comment:
Can we somehow parameterize the FOV angle so it isn't hardcoded everywhere?  Someday we may want to change the FOV angle dynamically and hard-coding it everywhere (especially in coded form like `22.5`) is going to make that difficult.